views/index.hbs

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+</head>
+<body>
+ <h1>hello data will be displayed here</h1>
+  {{#each}}
```

Comment:
Here you should specied what you want to iterate. In your case you want to iterate the data that you are sending when you render this page whta is call it `recipe`

So it will be like 
```
{{#each recipe }}
 <p>{{this.title}}</p>
{{/each}}
```
Inside, the `this` is the placeholder for each element of the `recipe`;